lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -28,7 +28,7 @@
 from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
 from xmodule.modulestore.tests.factories import ItemFactory, CourseFactory
 from xmodule.partitions.partitions import Group, UserPartition
-
+from openedx.core.djangoapps.self_paced.models import SelfPacedConfiguration
 
 class MasqueradeTestCase(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
```

Comment:
Add a newline above this. There should be two newlines before the class statement.